server/resources/UserResources.py

```diff
@@ -0,0 +1,60 @@
+from flask_restful import Resource, reqparse
+from models.UserModel import UserModel
+from flask_jwt_extended import (create_access_token, create_refresh_token, jwt_required, jwt_refresh_token_required, get_jwt_identity, get_raw_jwt)
+
+parser = reqparse.RequestParser()
+parser.add_argument('email', help = 'This field cannot be blank', required = True)
+parser.add_argument('password', help = 'This field cannot be blank', required = True)
+parser.add_argument('fname')
```

Comment:
Suggestion: camelcasing the name? ex: fName,

Reply:
Right, it should be fName and lName